jenkins/jenkins.yml

```diff
@@ -527,155 +561,7 @@ spec:
         yPos: 10
     description: ""
     name: Jenkins - High-Resolution
----
-apiVersion: influxdata.com/v2alpha1
```

Comment:
It looks like you removed one of your two Dashboards from this template, did you mean to do that? If so, you'll want to remove it from the list of provided resources in your README

Reply:
Whoops, good catch! I'll have to add that back in